junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -136,7 +149,20 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
 
         // at this point we can be sure that either no or exactly one WeldInitiator was found
         if (initiator == null) {
-            initiator = WeldInitiator.from(AbstractWeldInitiator.createWeld().addPackage(false, testInstance.getClass())).build();
+            Weld weld = createWeld();
+            Class<?> testClass = testInstance.getClass();
+            @SuppressWarnings("unchecked")
+            List<WeldCustomizer> customizers = (List<WeldCustomizer>) context.getRoot().getStore(Namespace.GLOBAL).get(WELD_CUSTOMIZERS);
+            if (customizers.isEmpty()) {
+                throw new IllegalStateException("Cannot build default Weld builder - no WeldCustomizer found");
```

Comment:
This should mention that neither customizer nor initiator field was found

Reply:
Good point. The absence of initiator field is kind of implicit here but it would be good to mention...